src/shared_modules/utils/msgDispatcher.h

```diff
@@ -32,8 +32,14 @@ namespace Utils
         , public RawValueDecoder
     {
         public:
-            MsgDispatcher()
-                : ThreadType{std::bind(&DispatcherType::dispatch, this, std::placeholders::_1)}
+            MsgDispatcher(const unsigned int threadPoolSize = std::thread::hardware_concurrency(),
+                          const size_t maxQueueSize = 0)
```

Comment:
Hi @Dwordcito

Maybe the `UNLIMITED_QUEUE_SIZE` macro could be moved to a common file and used here as well.